app/views/i18n/I18NEditModelView.coffee

```diff
@@ -128,54 +140,55 @@ module.exports = class I18NEditModelView extends RootView
     #- Enable patch submit button
 
     @$el.find('#patch-submit').attr('disabled', null)
+    @madeChanges = true
 
   onLanguageSelectChanged: (e) ->
+    if @madeChanges
+      return unless confirm('You have unsaved changes!')
     return if @hush
```

Comment:
Looks like this was already there, but while this variable name is cute, it's not super clear.